src/loader.ts

```diff
@@ -11,7 +11,7 @@ export async function loadCommands(): Promise<Collection<string, Command>> {
 	await Promise.all(
 		fs
 			.readdirSync("./dist/commands")
-			.filter((f) => f.endsWith(".js"))
+			.filter((f) => f.endsWith(".js") && !f.startsWith("sample"))
```

Comment:
create a function
`ts
function isActive(filename: String): bool {
    return f.endsWith(".js") && !f.startsWith("sample")
}
`
this way it can be reused and one only has to edit one thing instead of 3 places.

Reply:
did it